src/xtgeo/__init__.py

```diff
@@ -42,7 +41,7 @@ def _xprint(msg):
 except ImportError:
     ROXAR = False
 
-if not ROXAR:
+if platform.system() == "Linux" and not ROXAR:
```

Comment:
This is the only "new" thing here, aside from exporting symbols. Everything else is just some small consolidation or re-ordering